game_api.txt

```diff
@@ -347,13 +347,21 @@ The farming API allows you to easily register plants and hoes.
 
 ### Hoe Definition
 
-
 	{
-		description = "",                      -- Description for tooltip
-		inventory_image = "unknown_item.png",  -- Image to be used as wield- and inventory image
-		max_uses = 30,                         -- Uses until destroyed
-		material = "",                         -- Material for recipes
-		recipe = {                             -- Craft recipe, if material isn't used
+		-- Description for tooltip
+		description = "My Hoe",
+
+		-- Image to be used as wield- and inventory image
+		inventory_image = "unknown_item.png",
+
+		-- Uses until destroyed
+		max_uses = 30,
+
+		-- Material for recipes
+		material = "",
+
+		-- Craft recipe, if material isn't used
+		recipe = {
```

Comment:
The hoe definition changes above make the documentation longer and slightly less readable (because everything is so spaced out with comments inbetween), i cannot see any point in doing this.

Reply:
I did it because in my opinion it is easier to copy-paste the code and then remove those comments, and the text could be easier to read in a small text editor (it currently exceeds the 80 char limit).

I have changed it back in the latest commit.